README.md

```diff
@@ -1 +1,174 @@
-# Hello-Microverse-project-2
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>My README Template</b></h3>
+
+</div>
```

Comment:
- [ ] [OPTIONAL]Good job setting everything up 👍 , you made a great job, but there is still something you should improve in your README to make it more professional 😄 , Kindly remove the microverse logo , it could be better if you put yours 👍 or you delete it either